build_info/base-files.control

```diff
@@ -4,7 +4,6 @@ Version: @DEB_BASE-FILES_V@
 Architecture: @DEB_ARCH@
 Maintainer: @DEB_MAINTAINER@
 Replaces: dpkg (<< 1.20.9-2)
```

Comment:
`Conflicts` not `Replaces`

Reply:
So you want dpkg to get deleted 
fr

Reply:
And no it doesn't conflict

Reply:
> So you want dpkg to get deleted
> fr

No, that's what `Replaces` will do.

> And no it doesn't conflict

Yes it does, they both provide `/etc/dpkg/origin/procursus`

Reply:
Replaces: just allow overwriting files

Reply:
and I tested this
It works fine

Reply:
Then it should be both `Replaces` and `Conflicts`.

Reply:
This will prevent installation until dpkg is updated

Reply:
Also prevents downgrading dpkg

Reply:
Which I *do not* want to break.